make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import github
+import requests
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+WORKING_DIR = "./tmp"
+DIFF_FILE = "./diff.patch"
+
+auth = github.Auth.Token(os.getenv("GITHUB_TOKEN"))
```

Comment:
When you run the script without `GITHUB_TOKEN` set, it gives a not-so-helpful error message. Perhaps we can do `if 'GITHUB_TOKEN' in os.environ` or similar and print a helpful message and exit with a nonzero code instead?